test/rekt/pingsource_test.go

```diff
@@ -89,3 +90,18 @@ func TestPingSourceWithCloudEventData(t *testing.T) {
 
 	env.Test(ctx, t, pingsource.SendsEventsWithCloudEventData())
 }
+
+func TestPingSourceWithEventTypes(t *testing.T) {
+	t.Skip("The original test's logic is wrong. After it is fixed, this test wil start failing. So skip for now. This test will moved to experimental rekt package.")
```

Comment:
should the passing test be part of the pr?

not exactlty sure I get the motivation here.